.github/workflows/ProcessCommit.yml

```diff
@@ -0,0 +1,37 @@
+name: Process commit
+on: 
+  workflow_dispatch:
+  push:
+    branches:
+      - 'master'
+      - 'beta'
+      - 'release-*'
+    paths:
+      - 'dotnet/**'
+    ignore-paths:
+      - '*.md'
+
+jobs:
+  dispatchWorkflow:
+    runs-on: [ubuntu-latest]
+
+    steps:
+    - name: Dispatch Workflow
+      run: >
+        curl -X POST
+        -H 'Accept: application/vnd.github.v3+json' 
+        -H 'Authorization: token ${{ secrets.SECURE_TOKEN }}' 
+        --silent --show-error --fail 
+        ${{ secrets.BUILD_WORKFLOW_DISPATCH }} 
+        -d '{
+          "ref":"master",
+           "inputs": { 
+              "repository": "${{ github.repository }}",
+              "ref": "${{ github.ref }}",
+              "sha": "${{ github.sha }}",
+              "build-file": "dotnet\\DotNetStandardClasses.sln",
+              "package-version": ".\\.github\\generatePackageVersion.ps1",
```

Comment:
Pero... si usamos un ps para obtener la versión ¿no estamos habilitando a que cambien ese ps para ejecutar cualquier comando que me rompa la máquina de armado?

Reply:
We should use a CODEOWNERS file to check against this kind of changes

Reply:
Or expect the version label instead of how to calculate it (at least at first step). I would love to not have a list (that we will have to mantain).